c2corg_ui/static/js/auth/authservice.js

```diff
@@ -71,6 +71,30 @@ app.Authentication.prototype.isAuthenticated = function() {
   return !!this.userData && !this.isExpired_();
 };
 
+/**
+ * @return {number}
+ * @private
+ */
+app.Authentication.prototype.getUserId_ = function() {
```

Comment:
But if you want me to move the checkings to the protected-url-directive, where I will access authService, then I need to leave this function if I want to access the userId, I presume?